### PR TITLE
Release 5.3.0

### PR DESCRIPTION
**Pull request by iruzevic**
### Added

- New Integration for `Paycek payment`.

### CODE REVIEW

File: src/Integrations/Paycek/SettingsPaycek.php [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+								],
+							],
+							[
+								'component' => 'steps',
+								'stepsTitle' => \__('How to get the API key?', 'eightshift-forms'),
+								'stepsContent' => [
+									\__('Log in to your Paycek Merchant Account.', 'eightshift-forms'),
+									\__('Go to your Account.', 'eightshift-forms'),
+									\__('Go to "profile setting" and you will fin your Profile ID under "Code".', 'eightshift-forms'),
+									\__('Copy the Store ID into the field under the API tab or use the global constant.', 'eightshift-forms'),
```
Comment (piqusy) [inline src/Integrations/Paycek/SettingsPaycek.php:427]:
```suggestion
									\__('Copy the Profile ID into the field under the API tab or use the global constant.', 'eightshift-forms'),
```
?

**Review by piqusy — CHANGES_REQUESTED** [review]
Nice work 👏🏼 

Left a few suggestions

File: src/Rest/Routes/Integrations/Paycek/FormSubmitPaycekRoute.php [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+	{
+		return \rtrim(\strtr(\base64_encode($data), '+/', '-_'), '='); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_encode
+	}
+
+	/**
+	 * Generate payment URL for Paycek.
+	 *
+	 * @param string $profileCode can be found in profile settings (https://paycek.io).
+	 * @param string $secretKey can be found in profile settings (https://paycek.io).
+	 * @param string $paymentId unique payment id (id that you are using on your website to uniquely describe the purchase).
```
Comment (dadadavorin) [inline src/Rest/Routes/Integrations/Paycek/FormSubmitPaycekRoute.php:198]:
```suggestion
	 * @param string $paymentId unique payment ID (ID that you are using on your website to uniquely describe the purchase).
```

**Review by dadadavorin — APPROVED** [review]
Looks good after solving typos @piqusy noticed.

File: src/Rest/Routes/Integrations/Paycek/FormSubmitPaycekRoute.php [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+					break;
+			}
+
+			$output[$key] = $param;
+		}
+
+		$output['secretKey'] = UtilsSettingsHelper::getSettingsDisabledOutputWithDebugFilter(Variables::getApiKeyPaycek(), SettingsPaycek::SETTINGS_PAYCEK_API_KEY_KEY)['value'];
+		$output['profileCode'] = UtilsSettingsHelper::getSettingsDisabledOutputWithDebugFilter(Variables::getApiProfileKeyPaycek(), SettingsPaycek::SETTINGS_PAYCEK_API_PROFILE_KEY)['value'];
+		$output['language'] = UtilsSettingsHelper::getSettingValue(SettingsPaycek::SETTINGS_PAYCEK_LANG_KEY, $formId);
+		$output['paymentId'] = 'order_' . \time();
```
Comment (goranalkovic-infinum) [inline src/Rest/Routes/Integrations/Paycek/FormSubmitPaycekRoute.php:170]: Could this be a problem if in theory 2 orders come in at the exact same time?

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

File: src/Rest/Routes/Integrations/Paycek/FormSubmitPaycekRoute.php (hunk 3 above)
Comment (iruzevic) [reply src/Rest/Routes/Integrations/Paycek/FormSubmitPaycekRoute.php:170]: in theory yes

**Review by piqusy — CHANGES_REQUESTED** [review]

**Pull request merged**